### ISSUE TITLE
Wonky padding in header on mobile/ tablet breakpoints

### ISSUE THREAD

**Issue by akmadian** [creativecommons/cccatalog-frontend#775]
## Describe the bug

After design sprint 1 (#723) was merged, the header now has some wonky padding in mobile breakpoints
- The Search logo is pushed too far to the right
- The hamburger is not pushed off of the right side of the screen

## To Reproduce

1. Go to ccsearch-dev.creativecommons.org
2. Use browser to simulate a mobile/ tablet screen
3. See error

## Screenshots

![header0](https://user-images.githubusercontent.com/28791774/77197067-be1be700-6aa1-11ea-9429-697616c00be6.PNG)

## Desktop (please complete the following information)
 - OS: Win 10 Pro 1909
 - Browser: Firefox Dev. Ed. 75.0

**Comment by Tanuj22** [comment]
Would like to work on this

**Comment by akmadian** [comment]
go ahead @Tanuj22

**Comment by Tanuj22** [comment]
@akmadian The hamburger should be in the right corner?

**Comment by akmadian** [comment]
yep, right corner of the header

**Comment by abhisheknaiidu** [comment]
@akmadian, since it's labeled as the good first issue. So As per the new code contribution guidelines of CC, I just submitted a PR.

**Comment by Tanuj22** [comment]
Sorry I got busy in some personal work. @abhisheknaiidu You could have just informed me and submitted a PR for it.

**Comment by abhisheknaiidu** [comment]
@Tanuj22 Oh sorry for that, actually I didn't see any assigned person, so I thought that no one is working on this, and so I submitted a PR.

**Comment by akmadian** [comment]
@abhisheknaiidu Yep, please be sure to read through issue threads to see if anyone else has said that they are working on it before starting work. We may not always assign the issue to someone properly, but it was already marked as `in progress` so that should have been a pretty good indicator :/

We are ok with people starting on an issue without recieving approval from a maintainer (work on the issue itself has to be approved, but for someone to actually do that work does not need approval). We ask that people indicate in the issue thread that they are working on the issue, and that if someone has indicated that they are working on it, that others do not start work on it.

**Comment by abhisheknaiidu** [comment]
Yea @akmadian so sorry, I won't repeat again!

**Comment by kgodey** [comment]
This needs to be fixed in Vocabulary, transferring the issue there. @Tanuj22 @abhisheknaiidu please submit a PR there.
